oh_staff_ui/urls.py

```diff
@@ -5,7 +5,7 @@
     path("add_item/", views.add_item, name="add_item"),
     path("add_item/<int:parent_id>", views.add_item, name="add_item_from_parent"),
     path("item/<int:item_id>", views.edit_item, name="edit_item"),
-    path("", views.add_item),  # TODO: Change this to something better.....
```

Comment:
I changed the default route from `add_item` to `item_search`, which I'd meant to do for a while.  I expect users normally will want to start with a search when they log in, or switch from Admin to the main application.